packages/clerk-js/src/core/resources/User.ts

```diff
@@ -146,6 +154,40 @@ export class User extends BaseResource implements UserResource {
     return new ExternalAccount(json, this.path() + '/external_accounts');
   };
 
+  generateTOTP = async (): Promise<TOTPResource> => {
+    const json = (
+      await BaseResource._fetch<ExternalAccountJSON>({
+        path: '/me/totp',
+        method: 'POST',
+      })
+    )?.response as unknown as TOTPJSON;
+
+    return new TOTP(json);
+  };
+
+  verifyTOTP = async (code: string): Promise<TOTPResource> => {
```

Comment:
We should pass a params object here  to make it more future proof

```suggestion
  verifyTOTP = async (params: VerifyTOTPParams): Promise<TOTPResource> => {
```

Reply:
Yep